api/rs/slint/mcu.md

```diff
@@ -1,25 +1,25 @@
 # Slint on Microcontrollers (MCU)
 
-This document explains how to use Slint to develop a UI on a MCU in a bare metal environment.
+The following scetions explain how to use Slint to develop a UI on a MCU in a bare metal environment.
```

Comment:
```suggestion
The following sections explain how to use Slint to develop a UI in a bare metal environment.
```

Reply:
I fixed the typo, but I'm unsure about the removal of the MCU. The examples below refer to HAL crates that I think are clearly MCU material. How strong do you feel about it?

Reply:
not strongly